CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changes
 
 * [#202](https://github.com/clojure-emacs/orchard/issues/202): `orchard.inspect`: right-align indices when rendering indexed collections.
+* Internal: make `orchard.java.parser-utils/parse-java-test` more easily debuggable. 
```

Comment:
Such changes don't need a changelog entry. (I guess end users are not supposed to use this function directly)

Reply:
I normally never document internal changes unless they can be observed externally somehow (e.g. performance improvement). I think that's just noise for many of the users. At any rate - I don't feel strongly about this, so feel free to keep it.

Reply:
👍 Removing